cowboys/routers_cowboys.py

```diff
@@ -0,0 +1,32 @@
+from fastapi import APIRouter, Depends, Body, status
+from fastapi.responses import JSONResponse
+from sqlalchemy.orm import Session
+
+from cowboys import crud, schemas
+from settings import bd_coneccion
+
+router = APIRouter()
+
+@router.post ("/CrearCowboys", response_model = schemas.CrearCowboyRespuesta)
```

Comment:
Recordemos que las URLs de las API Rest estan orientadas a recursos.
Esto significa que cada URL represente un recurso y los metodos HTTP los que operan o modifican estos recurso.
Las APIs Rest no deberían tener URLs con acciones en el nombre, es decir, no deben llamarse como "CrearAlgo" o "BorrarAlgo" o "EditarAlgo"

En una API rest, ese "Algo" es el recurso, entonces solo vamos a tener una URL para ese recurso, por ejemplo:
 - `/algo`
 - `/cowboys`
 - `/titulos`

Las acciones sobre estos recursos o urls se corresponden con los metodos HTTP:
 - GET (obtener)
 - POST (guardar)
 - PUT (editar)
 - DELETE (borrar)

En este caso, la URL de crear deberia ser `POST /cowboy`.

```suggestion
@router.post ("/cowboys", response_model = schemas.CrearCowboyRespuesta)
```

Reply:
Mas info, si te interesa:
 - https://blog.hubspot.es/website/que-es-api-rest
 - https://es.wikipedia.org/wiki/Transferencia_de_Estado_Representacional

Tambien deben haber videos en youtube.

Reply:
entonces a todas las rutas las llamo iguales y se diferencian por el metodo HTTP